grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -79,6 +80,9 @@ export class RootBaseStore {
   @observable
   incidentsPage: any = this.initialQuery.p ? Number(this.initialQuery.p) : 1;
 
+  @observable
+  currentPage: { [key: string]: number } = {};
```

Comment:
we can use `src/pages/index.tsx`  -> `getMatchedPage` to get current page

Reply:
this gets you the page number, not the page string